x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/bulk_actions/validations.ts

```diff
@@ -101,6 +101,7 @@ export const validateBulkEditRule = async ({
 const istEditApplicableToImmutableRule = (edit: BulkActionEditPayload[]): boolean => {
   return edit.every(({ type }) =>
     [BulkActionEditTypeEnum.set_rule_actions, BulkActionEditTypeEnum.add_rule_actions].includes(
+      // @ts-expect-error upgrade typescript v4.9.5
       type
     )
   );
```

Comment:
It would be great to fix this error:

```ts
const istEditApplicableToImmutableRule = (edit: BulkActionEditPayload[]): boolean => {
  const applicableActions: BulkActionEditType[] = [
    BulkActionEditTypeEnum.set_rule_actions,
    BulkActionEditTypeEnum.add_rule_actions,
  ];
  return edit.every(({ type }) => applicableActions.includes(type));
};
```